.circleci/scripts/require-changelog.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+set -u
+set -o pipefail
+
+DEFAULT_BRANCH="origin/$(git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@')"
+if ( git diff -U0 "${DEFAULT_BRANCH}" -- package.json | grep '"version": "\d\\.\d\\.\d"' )
+then
+  ( git diff --name-only "${DEFAULT_BRANCH}" | grep CHANGELOG.md ) || exit 1
```

Comment:
Is it necessary to determine the "default branch"? I would think something like this would work:

```suggestion
if ( git diff -U0 HEAD^ -- package.json | grep '"version": "\d\\.\d\\.\d"' )
then
  ( git diff --name-only HEAD^ | grep CHANGELOG.md ) || exit 1
```

Reply:
Wait, no, this doesn't work 🤦‍♂️ This assumes the PR is just one commit.

Reply:
Good suggestion, that works! 9e8935a

Reply:
I wish there was a way to reference the target branch with CircleCI. It looks like this is not a thing, sadly ([though there is an open feature request for it](https://ideas.circleci.com/ideas/CCI-I-894)).

One more reason to switch to GitHub Actions I guess?